### ISSUE TITLE
Unable to load valid OAS 

### ISSUE THREAD

**Issue by undera** [chanced/openapi#5]
The following code fails with  message `Failed: json: cannot unmarshal object into Go struct field openapi.paths of type openapi.Example`. The spec is quite simple and valid, as far as I can see...
```go
func TestLoadData(t *testing.T) {
	data := `{
	  "openapi": "3.1.0",
	  "info": {
		"title": "",
		"version": "",
		"description": "Test file for loading pre-existing OAS"
	  },
	  "paths": {
		"/catalogue/{id}": {
		  "parameters": [
			{
			  "name": "id",
			  "in": "path",
			  "required": true,
			  "style": "simple",
			  "schema": {
				"type": "string"
			  },
			  "examples": {
				"an example": {
				  "value": "someval"
				}
			  }
			}
		  ]
		},
		"/catalogue/{id}/details": {
		  "parameters": [
			{
			  "name": "id",
			  "in": "path",
			  "style": "simple",
			  "required": true,
			  "schema": {
				"type": "string"
			  },
			  "example": "some-uuid-maybe"
			}
		  ]
		}
	  }
	}`

	var oas openapi.OpenAPI
	err := json.Unmarshal([]byte(data), &oas)
	if err != nil {
		t.Errorf("Failed: %s", err)
		t.FailNow()
	}
}
```

**Comment by chanced** [comment]
Ah, I accidentally had a `map[string]Example` field rather than `Examples` in a few places. This has been resolved and a new version (v0.0.6) has been pushed.

Thank you for the test. It made solving it quick!

**Comment by undera** [comment]
Thank you for fixing this!
